### PR TITLE
Jwoo/ios custom rendering part2

### PR DESCRIPTION
**Pull request by nesalang**
- added custom rendering
- added example of customizing inputnumber

### CODE REVIEW

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRegistration.h [hunk 1]
```diff
@@ -19,4 +20,5 @@
 
 - (void) setActionRenderer:(ACRBaseActionElementRenderer *)renderer cardElementType:(NSNumber *)cardElementType;
 
+- (void) setBaseCardElementRenderer:(ACRBaseCardElementRenderer *)renderer CardElementType:(ACRCardElementType)cardElementType;
```
Comment (bviglietta) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRegistration.h]: >CardElementType [](start = 75, length = 15)

lowercase: `cardElementType`

File: source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/CustomInputNumberRenderer.mm [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+@implementation CustomInputNumberRenderer
+
++ (CustomInputNumberRenderer *)getInstance
+{
+    static CustomInputNumberRenderer *singletonInstance = [[self alloc] init];
+    return singletonInstance;
+}
+
+//+ (ACRCardElementType)elemType;
```
Comment (bviglietta) [inline source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/CustomInputNumberRenderer.mm]: Remove commented-out code.

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRegistration.h (hunk 1 above)
Comment (nesalang) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRegistration.h]: thanks

---
In reply to: [171734434](https://github.com/Microsoft/AdaptiveCards/pull/1164#discussion_r171734434) [](ancestors = 171734434)

File: source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/CustomInputNumberRenderer.mm (hunk 2 above)
Comment (nesalang) [reply source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/CustomInputNumberRenderer.mm]: thanks

---
In reply to: [171734597](https://github.com/Microsoft/AdaptiveCards/pull/1164#discussion_r171734597) [](ancestors = 171734597)

**Pull request merged**